pkg/sync/syncer.go

```diff
@@ -290,84 +284,70 @@ func (s *syncer) SyncResources(ctx context.Context) error {
 		return nil
 	}
 
-	visited := make(map[subResource]struct{})
-	subResources := []subResource{{resourceTypeId: s.state.Current().ResourceTypeID}}
+	return s.syncResources(ctx)
```

Comment:
I think you need to return on l277 after you finish fetching all the resource types? It lets you checkpoint and it's possible there aren't actually any resource types so we wouldn't push any resource actions on the stack.

Reply:
Ah jeeze, I don't think I had this expanded all the way in the diff. I see that we do return. Disregard.